utils/spacewalk-utils.spec

```diff
@@ -20,90 +21,113 @@
 %{!?pylint_check: %global pylint_check 1}
 %endif
 
-%if 0%{?fedora} || 0%{?rhel}
-%global apache_group apache
-%endif
-%if 0%{?suse_version}
-%global apache_group www
-%endif
-
 Name:           spacewalk-utils
 Version:        4.1.5
 Release:        1%{?dist}
-Summary:        Utilities that may be run against a Uyuni server
+Summary:        Utilities that may be run against a SUSE Manager/Uyuni server
 License:        GPL-2.0-only AND GPL-3.0-or-later
 Group:          Productivity/Other
-
 Url:            https://github.com/uyuni-project/uyuni
 Source0:        https://github.com/spacewalkproject/spacewalk/archive/%{name}-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
-
+BuildRequires:  fdupes
+BuildRequires:  docbook-utils
+BuildRequires:  python3
 %if 0%{?pylint_check}
 BuildRequires:  spacewalk-python3-pylint
 %endif
-BuildRequires:  /usr/bin/docbook2man
-BuildRequires:  /usr/bin/pod2man
-BuildRequires:  docbook-utils
-BuildRequires:  python3
 BuildRequires:  uyuni-base-common
 
+%if 0%{?suse_version}
+Requires:       perl = %{perl_version}
+%else
+Requires:       perl(:MODULE_COMPAT_%(eval "`%{__perl} -V:version`"; echo $version))
+%endif
+# Required by debsolver.py
+Requires:       python3-PyYAML
+# Required by debsolver.py, cloneByDate.py
+Requires:       python3-uyuni-common-libs
+# Required by spacewalk-clone-by-date, spacewalk-sync-setup
+Requires:       salt
+# Required by cloneByDate.py, spacewalk-clone-by-date
+Requires:       spacewalk-backend
+# Required by cloneByDate.py
+Requires:       spacewalk-backend-sql
+# Required by spacewalk-final-archive
+Requires:       spacewalk-backend-tools >= 2.2.27
```

Comment:
Is spacewalk-final-archive in the "supported" area?
If not, this dependency is not needed here in this package.

Reply:
Right. Seems a small typo... either at the comment or at the dependency itself. Will review tomorrow.

Reply:
The comment was wrong. Fixed now (not pushed yet):
```
# Required by cloneByDate.py, depsolver.py 
Requires:       spacewalk-backend-tools >= 2.2.27